okhttp-clients/src/test/java/com/palantir/remoting3/okhttp/metrics/HostMetricsRegistryTest.java

```diff
@@ -29,8 +29,8 @@ public void test() {
         HostMetricsRegistry hostRegistry = new HostMetricsRegistry(registry, "service");
 
         // TODO(jellis): check different meter when host is added as a tag
```

Comment:
do you want to add the test for alternate host tag?

Reply:
yeah the test was right. removed the todo